teams/alumni.toml

```diff
@@ -21,6 +21,7 @@ members = [
     "dwijnand",
     "edunham",
     "erickt",
+    "fmoko",
```

Comment:
There is no need to add you explicitly to the alumni team, being in `alumni = ["..."]` inside a team automatically puts you here as well.